src/sections/Newsletter.js

```diff
@@ -72,26 +115,61 @@ class Newsletter extends Component {
                 </Col>
                 <Col xl={12}>
                     <Division>
-                        <Col xs={11} className='mx-auto'>
-
-                            <Text
-                                weight='bold'
-                                color={theme.colors.white}
-                                size={theme.sizes.regular}
-                            >
-                                {NEWSLETTER.description}
-                            </Text>
-                            <Wrapper md={4}>
-                                <Form.Control id='emailHolder' size='sm' type='text' placeholder={NEWSLETTER.placeholder} />
-                            </Wrapper>
-                            <Wrapper md={2}>
-                                <Button variant='info' onClick={() => postData()}>
-                                    {NEWSLETTER.buttonText}
-                                </Button>
-                            </Wrapper>
-
-
-                        </Col>
+                        {this.state.submitted
+                            ? (
+                                <Col xs={11} className='mx-auto'>
+                                    <Text
+                                        weight='bold'
+                                        color={theme.colors.white}
+                                        size={theme.sizes.regular}
+                                    >
+                                    Thanks for suscribing!
+                                    </Text>
+                                    <Text
+                                        weight='bold'
+                                        color={theme.colors.white}
+                                        size={theme.sizes.regular}
+                                    >
+                                        {NEWSLETTER.description}
+
+                                    </Text>
+
+
+                                </Col>
+                            )
+                            : (
+                                <Col xs={11} className='mx-auto'>
+
+                                    <Text
+                                        weight='bold'
+                                        color={theme.colors.white}
+                                        size={theme.sizes.regular}
+                                    >
+                                        {NEWSLETTER.description}
+                                    </Text>
+                                    <Form>
```

Comment:
Right so you want to move your onSubmit/onClick from the Button to the Form right here